src/main/scheduler/scheduler.c

```diff
@@ -202,10 +202,10 @@ void getCheckFuncInfo(cfCheckFuncInfo_t *checkFuncInfo)
 void getTaskInfo(taskId_e taskId, taskInfo_t * taskInfo)
 {
     taskInfo->isEnabled = queueContains(getTask(taskId));
-    taskInfo->desiredPeriodUs = getTask(taskId)->id->desiredPeriodUs;
-    taskInfo->staticPriority = getTask(taskId)->id->staticPriority;
-    taskInfo->taskName = getTask(taskId)->id->taskName;
-    taskInfo->subTaskName = getTask(taskId)->id->subTaskName;
+    taskInfo->desiredPeriodUs = getTask(taskId)->attr->desiredPeriodUs;
+    taskInfo->staticPriority = getTask(taskId)->attr->staticPriority;
+    taskInfo->taskName = getTask(taskId)->attr->taskName;
+    taskInfo->subTaskName = getTask(taskId)->attr->subTaskName;
```

Comment:
yes, as noted in slack, these lines read so much better now.  Previously the code was getting a task by an id (integer) and then de-referencing something that was a different 'id', which was not an integer, to get some struct member.